### PR TITLE
SPCTR-1810 - Fix: TOC - Adding 'uagb-toc-hide-heading' class to exclude heading from TOC list not working on front-end.

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
Adding 'uagb-toc-hide-heading' class to exclude heading in TOC list not working on front-end.

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Testing Evidence - https://share.getcloudapp.com/eDu7wrjX

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: src/blocks/table-of-contents/edit.js [hunk 1]
```diff
@@ -294,7 +294,17 @@ export default compose(
 		} else {
 			headerArray = document.body.getElementsByClassName( 'is-root-container' )[0]?.querySelectorAll( 'h1, h2, h3, h4, h5, h6' );
 		}
-
+		const excludeBlock = document.querySelectorAll( '.uagb-toc-hide-heading' );
+		if ( excludeBlock !== 'undefined' ) {
```
Comment (RajkiranBagal) [inline src/blocks/table-of-contents/edit.js:298]: Can we just check like this? - `if ( excludeBlock ) { `

File: src/blocks/table-of-contents/edit.js [hunk 2]
```diff
@@ -294,7 +294,17 @@ export default compose(
...
+			excludeBlock.forEach( function ( heading ) {
+				const innerHeading = heading.querySelectorAll(  'h1, h2, h3, h4, h5, h6' );
+				if ( innerHeading !== 'undefined' ) {
```
Comment (RajkiranBagal) [inline src/blocks/table-of-contents/edit.js:301]: Can we just check like this? - `if ( innerHeading ) { `

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/table-of-contents/edit.js (hunk 1 above)
Comment (supriyajjadhav) [reply src/blocks/table-of-contents/edit.js]: Updated condition.

File: src/blocks/table-of-contents/edit.js (hunk 2 above)
Comment (supriyajjadhav) [reply src/blocks/table-of-contents/edit.js]: Updated condition.

File: README.md [hunk 3]
```diff
@@ -175,6 +175,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Icon List - Text Alignment not working properly on front-end.
 * Fix: Form - Backward Compatibility Stylings Issue on the editor.
 * Fix: Customizer was frozen when theme blog settings were changed.
+* Fix: TOC - Adding 'uagb-toc-hide-heading' class to exclude heading from TOC list not working on front-end.
```
Comment (RajkiranBagal) [inline README.md:178]: TOC -> Table of Contents

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (supriyajjadhav) [reply README.md]: Updated

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**